src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/Get-AzureRmApplicationInsights.md

```diff
@@ -0,0 +1,132 @@
+---
+external help file: Microsoft.Azure.Commands.ApplicationInsights.dll-Help.xml
+Module Name: AzureRM.ApplicationInsights
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmApplicationInsights
+
+## SYNOPSIS
+Get application insights resources
+
+## SYNTAX
+
+### ResourceGroupParameterSet
+```
+Get-AzureRmApplicationInsights [[-ResourceGroupName] <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ComponentNameParameterSet
+```
+Get-AzureRmApplicationInsights [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzureRmApplicationInsights [-ResourceId] <ResourceIdentifier> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get application insights resources in a resource group or specific resource
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmApplicationInsights -ResourceGroupName "testgroup" -Name "test"
```

Comment:
@frankguodongchen for all examples in your markdown files, please include a sample of what the output will look like for the given cmdlet call